### PR TITLE
[trivial] Fix error messages in CFeeBumper

### PR DESCRIPTION
**Pull request by kallewoof**
…pool fee rate.

### CODE REVIEW

File: src/wallet/feebumper.cpp [hunk 1]
```diff
@@ -196,7 +196,13 @@ CFeeBumper::CFeeBumper(const CWallet *pWallet, const uint256 txidIn, const CCoin
     // moment earlier. In this case, we report an error to the user, who may use totalFee to make an adjustment.
     CFeeRate minMempoolFeeRate = mempool.GetMinFee(gArgs.GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000);
     if (nNewFeeRate.GetFeePerK() < minMempoolFeeRate.GetFeePerK()) {
-        vErrors.push_back(strprintf("New fee rate (%s) is less than the minimum fee rate (%s) to get into the mempool. totalFee value should to be at least %s or settxfee value should be at least %s to add transaction.", FormatMoney(nNewFeeRate.GetFeePerK()), FormatMoney(minMempoolFeeRate.GetFeePerK()), FormatMoney(minMempoolFeeRate.GetFee(maxNewTxSize)), FormatMoney(minMempoolFeeRate.GetFeePerK())));
+        vErrors.push_back(strprintf(
+            "New fee rate (%s) is lower than the minimum fee rate (%s) to get into the mempool. "
+            "The totalFee value should be at least %s or the settxfee value should be at least %s to add transaction.",
```
Comment (promag) [inline src/wallet/feebumper.cpp]: Nit, there are only 2 error messages with periods, this one and:
https://github.com/bitcoin/bitcoin/blob/2c83c511dcda98550b660f6b62da7fab8548ca5c/src/wallet/feebumper.cpp#L293
How about removing the period in the last one and here: `"New fee rate ... - the totalFee value should ... to add transaction"`

**Review by promag — COMMENTED** [review]
Concept ACK.

Nit, reword PR and commit to `Fix error messages in CFeeBumper`.

File: src/wallet/feebumper.cpp (hunk 1 above)
Comment (kallewoof) [reply src/wallet/feebumper.cpp]: Sounds good -- done.

Comment (promag) [reply src/wallet/feebumper.cpp]: My suggestion included replacing the inner period with -

**Comment by TheBlueMatt** [comment]
utACK a16340758e3aa2a73a8b594eeb5fae7f81f9eea1

File: src/wallet/feebumper.cpp [hunk 2]
```diff
@@ -284,7 +290,7 @@ bool CFeeBumper::commit(CWallet *pWallet)
         // along with an exception. It would be good to return information about
         // wtxBumped to the caller even if marking the original transaction
         // replaced does not succeed for some reason.
-        vErrors.push_back("Error: Created new bumpfee transaction but could not mark the original transaction as replaced.");
+        vErrors.push_back("Error: Created new bumpfee transaction but could not mark the original transaction as replaced");
```
Comment (promag) [inline src/wallet/feebumper.cpp]: BTW, is there a reason to have these `Error: ` prefixes?

**Review by promag — COMMENTED** [review]
utACK a163407.

File: src/wallet/feebumper.cpp (hunk 2 above)
Comment (kallewoof) [reply src/wallet/feebumper.cpp]: Good point. Seems to be used only in a few places. I am going to remove the `Error:` prefix in the 3 locations it's used.

File: src/wallet/feebumper.cpp [hunk 3]
```diff
@@ -267,15 +273,15 @@ bool CFeeBumper::commit(CWallet *pWallet)
     CValidationState state;
     if (!pWallet->CommitTransaction(wtxBumped, reservekey, g_connman.get(), state)) {
         // NOTE: CommitTransaction never returns false, so this should never happen.
-        vErrors.push_back(strprintf("Error: The transaction was rejected! Reason given: %s", state.GetRejectReason()));
+        vErrors.push_back(strprintf("The transaction was rejected! Reason given: %s", state.GetRejectReason()));
```
Comment (promag) [inline src/wallet/feebumper.cpp]: :trollface:nit, follow format below (replace `!` by `:`), check others.

**Review by promag — COMMENTED** [review]
utACK 28eabb0.

File: src/wallet/feebumper.cpp (hunk 3 above)
Comment (kallewoof) [reply src/wallet/feebumper.cpp]: lol.. fixed, thanks.

**Comment by jonasschnelli** [comment]
utACK a02c5e459a1e27e3c2ca7206ee703bccca126ff9

**Comment by fanquake** [comment]
utACK a02c5e4

**Pull request merged**